turbo.json

```diff
@@ -1,43 +1,36 @@
 {
-    "$schema": "https://turbo.build/schema.json",
-    "globalDependencies": [
-        "**/.env.*local",
-        "**/.env.local",
-        "**/.env.*",
-        "**/.env"
-    ],
-    "globalEnv": [
-        "NODE_ENV",
-        "JWT_SECRET",
-        "HMS_MANAGEMENT_TOKEN",
-        "HMS_TEMPLATE_ID"
-    ],
-    "pipeline": {
-        "build": {
-            "dependsOn": [
-                "^build",
-                "^db:generate",
-                "^db:push"
-            ],
-            "outputs": [
-                ".next/**",
-                "!.next/cache/**"
-            ]
-        },
-        "lint": {},
-        "dev": {
-            "dependsOn": [
-                "^dev",
-                "^db:generate"
-            ],
-            "cache": false,
-            "persistent": true
-        },
-        "db:generate": {
-            "cache": false
-        },
-        "db:push": {
-            "cache": false
-        }
+  "$schema": "https://turbo.build/schema.json",
+  "globalDependencies": [
+    "**/.env.*local",
+    "**/.env.local",
+    "**/.env.*",
+    "**/.env"
+  ],
+  "globalEnv": [
+    "NODE_ENV",
+    "JWT_SECRET",
+    "HMS_MANAGEMENT_TOKEN",
+    "HMS_TEMPLATE_ID"
+  ],
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build", "^db:generate", "^db:push", "^db:scrape"],
+      "outputs": [".next/**", "!.next/cache/**"]
+    },
+    "lint": {},
+    "dev": {
+      "dependsOn": ["^dev", "^db:generate"],
+      "cache": false,
+      "persistent": true
+    },
```

Comment:
Now that I think about it, so people aren't confused when developing locally, let's add the seed/scrape to this task (dev) as well.

Thanks! 🚀